docs/operator-manual/applicationset/Generators-SCM-Provider.md

```diff
@@ -189,15 +189,17 @@ spec:
       - repositoryMatch: ^myapp
         pathsExist: [kubernetes/kustomization.yaml]
         labelMatch: deploy-ok
-      # ... OR any repository starting with "otherapp" AND a Helm folder.
+      # ... OR include any repository starting with "otherapp" AND a Helm folder and doesn't have file disabledrepo.txt.
       - repositoryMatch: ^otherapp
         pathsExist: [helm]
+        pathsDoesntExist: [disabledrepo.txt]
   template:
   # ...
 ```
 
 * `repositoryMatch`: A regexp matched against the repository name.
 * `pathsExist`: An array of paths within the repository that must exist. Can be a file or directory, but do not include the trailing `/` for directories.
+* `pathsDoesntExist`: An array of paths within the repository that must not exist. Can be a file or directory, but do not include the trailing `/` for directories.
```

Comment:
Is it possible to have a check for trailing "/" and trim it while storing the value or checking for path instead?